data/weapons.txt

```diff
@@ -796,20 +796,20 @@ outfit "Torpedo Launcher"
 		"hit effect" "large explosion"
 		"die effect" "tiny explosion"
 		"inaccuracy" 5
-		"velocity" 7
+		"velocity" 3.5
```

Comment:
In general I prefer to have a projectile's starting velocity match its long-term velocity for the sake of the firing AI's range calculation. Is there a particular reason to make this lower?